design-system/src/_includes/pattern-library/foundations/foundations-color/color.html

```diff
@@ -1,10 +1,236 @@
-<h3>White and greys</h3>
+<h3>Brand</h3>
 <div class="coop-l-grid">
     <div class="coop-l-grid__item coop-l-grid__item--small-6 coop-l-grid__item--medium-4">
         <div class="dm-colour">
-            <div class="dm-colour__swatch dm-colour__swatch--wide coop-u-white-bg coop-u-border">
+            <div class="dm-colour__swatch dm-colour__swatch--wide coop-u-brand-bg coop-u-padding-q coop-u-margin-resp-b-8-4">
+                <h4 class="coop-u-white">Co-op blue</h4>
+                <ul class="list-bare dm-colour__definition coop-u-white">
```

Comment:
Should the `list-bare` class be ~~`coop-u-list--bare`~~ `coop-list-bare`? There are a few other instances further down.